questions/192/explanation.md

```diff
@@ -4,10 +4,9 @@ In particular, in the expression `g(f1(), f2())`,  we don't know whether `f1` or
 
 Also: There isn't even a requirement on the implementation that `f1` and `f2` gets evaluated *in the same order each time*. So after calling `h` twice, `v` can contain `1212`, `2121`, `1221` or `2112`.
 
-§[expr.call]¶5: in the C++ standard:
+§[expr.call]¶7:
+> The initialization of a parameter, including every associated value computation and side effect, is indeterminately sequenced with respect to that of any other parameter.
 
-"The initialization of a parameter, including every associated value computation and side effect, is indeterminately sequenced with respect to that of any other parameter."
+And the helpful note in §[intro.executation]¶note-5:
```

Comment:
```suggestion
And the helpful note in §[intro.execution]¶note-5:
```

Reply:
Hah, good catch. Seems like it's been like that since the question was authored.